src/views/Home.vue

```diff
@@ -17,17 +17,19 @@
 import { mapState } from 'vuex'
 
 export default {
-  head: {
-    title: {
-      inner: 'Home'
-    },
-    meta: [
-      {
-        name: 'description',
-        content: 'Bento-starter home page',
-        id: 'desc'
-      }
-    ]
+  head: function() {
+    return {
+      title: {
+        inner: 'Home'
+      },
+      meta: [
+        {
+          name: 'description',
+          content: `${this.appTitle} home page`,
```

Comment:
Did you check that it's working correctly ?

Reply:
I checked what is generated when I build the app. In the dist I have the right head content. I checked also what is in the head content when I start a server. All is OK to me.